Monika After Story/game/script-topics.rpy

```diff
@@ -14195,3 +14195,28 @@ label monika_translating_poetry:
         m 5ekbsa "It would be nice if you could read some of them for me sometime..."
         m 5rkbsu "You'd have to translate them for me first, though~"
     return
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_diving",
+            category=["nature"],
+            prompt="scuba diving",
+            random=True
+        )
+     )
+label monika_diving:
+    m 3eub "[player], have you heard about scuba diving?"
+    m 3eub "after immersion in water, you can see a wonderful underwater world full of diverse marine life."
+    m 7eub "There you will meet:schools of fish, coral reefs, and sea greens,jellyfish's."
+    m 4eub "There may also be sharks."
+    m 2eud "As a rule, they are not in special places for diving."
+    m 7eud "And if there is,then be careful."
```

Comment:
```suggestion
    m 7eud "And if there are, then be careful."
```
I think it goes without saying that you should be careful if you meet a shark, and I might actually skip this line and go straight to Monika giving you advice on what to do, such as staying calm.